recipes/xcb-proto/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.12" %}
+{% set md5 = "14e60919f859560f28426a685a555962" %}
+
+package:
+  name: xcb-proto
+  version: {{ version }}
+
+source:
+  fn:  xcb-proto-{{ version }}.tar.bz2
+  url: http://xcb.freedesktop.org/dist/xcb-proto-{{ version }}.tar.bz2
+  md5: {{ md5 }}
+  patches:
+    # This patch makes "make check" to work
+    # Taken from http://www.linuxfromscratch.org/patches/blfs/svn/xcb-proto-1.12-schema-1.patch
+    - xcb-proto-1.12-schema-1.patch
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - python 2.7*
```

Comment:
Remove the version here and add py3k on the skip above. `skip: true  # [win or osx or py3k]` should work.